src/services/network.js

```diff
@@ -22,13 +23,30 @@ const NETWORKS = [
       aph: 'https://testnet.aphelion-neo.com:62443/api',
       net: 'TestNet',
       rpc: 'https://testneo.aphelion-neo.com:20331',
+      fee: 0,
     },
   },
 ];
 
+const NETWORK_FEE_OPTIONS = [
```

Comment:
This is fine for now still, as it is what we have on desktop, but we really need to support setting fee per K/Byte and support any value instead of powers of 10.